CHANGELOG.md

```diff
@@ -9,9 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-
 ### Changed
 
+- `TargetConfigs` now allows unexpected fields again because block attributes were not being saved - [#60](https://github.com/PrefectHQ/prefect-dbt/pull/60)
```

Comment:
```suggestion
- `TargetConfigs` now allows unexpected fields again because block attributes were not being saved - [#64](https://github.com/PrefectHQ/prefect-dbt/pull/64)
```